packages/SwingSet/package.json

```diff
@@ -77,7 +77,8 @@
     "files": [
       "test/**/test-*.js"
     ],
-    "timeout": "20m"
+    "timeout": "20m",
+    "workerThreads": false
```

Comment:
After this lands, I'd be curious to remove this and see if it affects anything. If "workerThreads" share a JS heap, then we'll probably need to keep it disabled. But maybe they only share a unix process memoryspace, and still have separate instances of the V8 engine (and thus separate JS heaps, so their GC doesn't affect each other), in which case we could use them.